scen_def/SXC_IsarsCrossfire.cfg

```diff
@@ -341,6 +364,9 @@
       [/not]
       [then]
         {SXC_ENEMY_LEADER_3 3 11 "Death Knight" 9 2434 8 49 9 52 70 RA BLANK BLANK BLANK}
+        {SXC_ENEMY_LEADER_ADD_ABILITY 9 {ABILITY_SXC_REGENERATESX 250}}
+        {SXC_ENEMY_LEADER_SET_SPECIALS 9 0 append {WEAPON_SPECIAL_SXC_VIGILANCE} {BLANK}}
+        {SXC_ENEMY_LEADER_SET_SPECIALS 9 2 append {WEAPON_SPECIAL_SXC_PRECISIONFOCUS} {BLANK}}
```

Comment:
I'd be tempted to make an event with `name=respawn_death_knight` and use `[fire_event]` instead of multiple copies.